CRM/Contribute/Form/SoftCredit.php

```diff
@@ -108,16 +108,15 @@ public static function buildQuickForm(&$form) {
   /**
    * Add PCP fields for the new contribution form and others.
    *
-   * @param CRM_Core_Form &$form
+   * @param CRM_Core_Form $form
    *   The form being built.
    * @param string $suffix
    *   A suffix to add to field names.
    */
-  public static function addPCPFields(&$form, $suffix = '') {
+  public static function addPCPFields($form, $suffix = '') {
```

Comment:
Are you sure about this? Doesn't it need to be passed by reference?

Reply:
@mattwire yeah pass-by-ref is meaningless for objects (maybe that wasn't true in php 5.1 when they added it)